addons/packages/grafana/7.5.16/README.md

```diff
@@ -0,0 +1,25 @@
+# grafana Package
+
+This package provides << awesome functionality >> using [grafana](https://INFO_NEEDED).
+
+## Components
+
+## Configuration
+
+The following configuration values can be set to customize the grafana installation.
+
+### Global
+
+| Value | Required/Optional | Description |
+|-------|-------------------|-------------|
+| `name` | Optional | description |
+
+### grafana Configuration
+
+| Value | Required/Optional | Description |
+|-------|-------------------|-------------|
+| `name` | Optional | description |
+
+## Usage Example
+
+The follow is a basic guide for getting started with grafana.
```

Comment:
This entire README.md needs to be updated to include, at a minimum, the content that is provided in the 7.5.11 version of the package.

Reply:
thx, I will updated it later